www/docs/guides/walletAccount.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_position: 9
+---
+
+# WalletAccount
+
+**Use wallets (like Braavos & ArgentX) to sign your transactions in your DAPP.**
+
+The `WalletAccount` class is similar to the regular `Account` class, but is also able to ask to a browser wallet to sign and send a transaction. Some other cool functionalities will be detailed hereunder.
+
+The private key of a WalletAccount is held in a browser wallet (as ArgentX or Braavos), and any signature is managed by the wallet. You don't have to manage in your DAPP the security of any private key.
+
+:::caution
+This class is working only in the scope of a DAPP. You can't use it in a node.js script.
```

Comment:
The intention is to be used in the browser environment, but technically you could build a node wallet service using SWO *wallet-api <-> some-service (whatever app) that could act as a bridge to a mobile wallet or some signing service and connect the node app to it.